### ISSUE TITLE
Stuck on "Continue without premium"

### ISSUE THREAD

**Issue by zingmars** [wabbajack-tools/wabbajack#1007]
When trying to install a modpack:

![image](https://user-images.githubusercontent.com/917126/89434540-51ea0900-d74c-11ea-9987-68a8296815b8.png)

I might be missing something but there's no way to actually accept. And is there a particular reason why you have to accept instead of just giving a warning that it might take forever?

**Comment by zingmars** [comment]
Nevermind, I decided to install things manually with Vortex since the UX of this program is kinda terrible all around.

**Comment by halgari** [comment]
Well if you ever want help debugging this and can wait for longer than 15 minutes, I'll be glad to take a look at your log file and help you debug it.

**Comment by zingmars** [comment]
[Wabbajack.current.log](https://github.com/wabbajack-tools/wabbajack/files/5029631/Wabbajack.current.log) If you want to look at it, but there doesn't seem to be anything wrong. It just that there was no UI prompt to get it to continue.

**Comment by halgari** [comment]
Thanks I'll take a look sometime soon. Honestly the non-premium paths are a bit hard to debug because the Nexus forbids having multiple accounts for a single human, and so I can't test the non-Premium code, because I have premium, as such the experience is less than optimal. I'll look into it sometime soon. 

That also being said, $4 for premium for a month is well worth the price.

**Comment by zingmars** [comment]
That's fair, I just don't really mod my games often enough to generally justify it.

Fwiw, I tried restarting 3 times, and the same thing happened. Also when I went back to home screen and to the mod view, it said "awaiting input" in the corner, but there was no way to return to the installation view (the install icon was grayed out) and that basically forced me to restart the app.

**Comment by zingmars** [comment]
Hi, figured I opened an issue so I might as well just add some more info to it.

The issue seems to be random, it will happen sometimes, but now that I tried a couple more times the message does seem to pop up properly. I did find, however, a way to bug the prompt 100% of the time by opening the settings menu before clicking the button. It might be a different issue, or it might be related. Here's a video (skip to 1:00) - https://ayy.lv/selif/ktxzcbrm.mp4. The video also displays being stuck in the "awaiting input" state with no way to actually return to the view (or at least I didn't find any. I _could_ be blind).

Lastly, is there a way to disable the automatic opening of README when you open a modlist? I can understand that happening the first time you open it, but afterwards it just becomes an annoyance.
